bigquery_etl/cli/query.py

```diff
@@ -20,9 +20,40 @@
 
 
 QUERY_NAME_RE = re.compile(r"(?P<dataset>[a-zA-z0-9_]+)\.(?P<name>[a-zA-z0-9_]+)")
+QUERY_FILE_RE = re.compile(
```

Comment:
Using "sql file" rather than "query file" terminology would be more precise. I was interpreting "query file" as "query.sql" specifically.